test/evergreen.yml

```diff
@@ -682,6 +682,9 @@ tasks:
       - func: "fetch artifacts"
       - func: "compile wiredtiger"
       - func: "make check all"
+        vars:
+          smp_command: -j 1
```

Comment:
Is there any special reason the smp number is set to 1 here? The default value is the number of cpus of the building host.

Reply:
Yeah because if we run `make-check-test` with ASAN which is the intention it can't run on multiple CPUs as some of the tests begin failing. So to make sure it has as much resources per test I limit it to 1 cpu. If its run not under ASAN then it can run on multiple CPUs.

So the options are:
- Add `-j 1`
- Duplicate the task and make an asan specific one

Reply:
Maybe we can avoid diverging the `make-check-test` task by extending func `make check all` with an argument (to specific smp number)? 

Adding a comment to describe the reason for setting `-j 1`.

Reply:
It has an argument for a specific `smp` number however the smp number on the variants isn't actually 1 so I couldn't just pass it from the variant level.